src/Ocean/HydrostaticBoussinesq/HydrostaticBoussinesqModel.jl

```diff
@@ -276,8 +279,11 @@ applies convective adjustment in the vertical, bump by 1000 if ∂θ∂z < 0
 - `∂θ∂z`: value of the derivative of temperature in the z-direction
 """
 @inline function diffusivity_tensor(m::HBModel, ∂θ∂z)
-    ∂θ∂z < 0 ? κ = (@SVector [m.κʰ, m.κʰ, 1000 * m.κᶻ]) : κ =
-        (@SVector [m.κʰ, m.κʰ, m.κᶻ])
+    ∂θ∂z < 0 ? κ = (@SVector [m.κʰ, m.κʰ, m.κᶜ]) : κ = (@SVector [
+        m.κʰ,
+        m.κʰ,
+        m.κᶻ,
+    ])
```

Comment:
@simonbyrne @charleskawczynski any ideas on how I can trick the formatter into making this look nicer? the previous version is much easier to read

Reply:
You can [turn off the formatter](https://domluna.github.io/JuliaFormatter.jl/stable/skipping_formatting/) for chunks for code. We do this in a [handful of places already](https://github.com/CliMA/ClimateMachine.jl/search?q=format%3A+off&unscoped_q=format%3A+off)

Reply:
Or break it down into multiple lines with temporary arrays?

Reply:
realized I only needed to change the third component of the array and I can just use a temporary variable for that and make the actual array/matrix in the `return` statement